andela/wg-baqi
# #166466091 Configures Middleware

#### Description
Configures middlewares correctly as per Django 2 rules
#### Type of change
- [x] Currently, CSS is not being processed despite having configured whitenoise for this.
- [x] This bug is being caused by outdated middleware configuration
- [x] The PR fixes the bug
#### How Has This Been Tested?
- Clone [this repo](https://github.com/andela/wg-baqi.git) to your local computer
- CD into `wger` and create a virtual env by doing `python3 -m virtualenv venv`
- Activate the virtual env: `source venv/bin/activate` && install requirements `pip install -r requirements.txt`
- Run migrations by invoking the `bootstrap-wger` command as is on the Readme.rst
- Checkout to the `bg-middleware-bug-166466091` branch
- In your browser login as an admin by going to `http://127.0.0.1:8000/en/admin/` 
- If you are already logged in as an admin, you should be able to navigate to `/en/user/api-key/` to get an api key
#### Any background context:
Current configurations will only work if you are logged in as an admin
#### Story ID
[#166466091](https://www.pivotaltracker.com/story/show/166466091)
#### Screenshots
<img width="1432" alt="Screen Shot 2019-06-07 at 05 37 06" src="https://user-images.githubusercontent.com/38868860/59078130-4f8ce480-88e6-11e9-9d32-240a9b18d7a2.png">
